docs/production/cloudcaptain.mdx

```diff
@@ -13,7 +13,7 @@ CloudCaptain comes with **first-class support for JHipster** and works by creati
 
 :::tip
 
-As an alternative to CloudCaptain you can also deploy your JHipster application to AWS using [Elastic Beanstalk](/aws).
```

Comment:
We dropped support for AWS in v8.0.0. https://www.jhipster.tech/2023/11/02/jhipster-release-8.0.0.html